dask/tests/test_distributed.py

```diff
@@ -556,3 +556,32 @@ async def test_annotation_pack_unpack(c, s, a, b):
     unpacked_hlg = HighLevelGraph.__dask_distributed_unpack__(packed_hlg)
     annotations = unpacked_hlg["annotations"]
     assert annotations == {"workers": {"n": ("alice",)}}
+
+
+@gen_cluster(client=True)
```

Comment:
The `distributed` mindeps build is failing because this is the last test in this module (I remember we ran into this issue at some point before). Could you move this up so it's not the last test